backend/tournesol/urls.py

```diff
@@ -70,6 +74,11 @@
         ContributorRatingList.as_view(),
         name="ratings_me_list",
     ),
+    path(
+        "users/me/contributor_ratings/_all/",
```

Comment:
Do we need this route? Can't we just use the patch on the route `"users/me/contributor_ratings/"` and rely on the ContributorRatingList view?

Reply:
I actually hesitated with this solution. Implementing `PATCH` on a collection of resources does not seem to be a very common pattern in RESTful APIs. And DRF mixins are not suitable for this case, so a custom implementation would be necessary anyway.

I also find that defining a special identifier makes the intent a bit more explicit: `_all` can be PATCHed just like any other instance. And any future action/route on individual ratings could also be defined for this identifier too.

But it's mostly a matter of taste. If you prefer to stick closer to "standard" REST I can change it.

Reply:
Sounds good! Let's go with your initial proposition 👍